src/Monolog/Handler/BrowserConsoleHandler.php

```diff
@@ -175,7 +176,7 @@ private static function handleStyles(string $formatted): array
             $args[] = '"font-weight: normal"';
 
             $pos = $match[0][1];
-            $format = substr($format, 0, $pos) . '%c' . $match[1][0] . '%c' . substr($format, $pos + strlen($match[0][0]));
+            $format = Utils::substr($format, 0, $pos) . '%c' . $match[1][0] . '%c' . Utils::substr($format, $pos + Utils::strlen($match[0][0]));
```

Comment:
These should be undone as the $pos offset is in bytes and not unicode characters

Reply:
Utils::substr work with bytes count now